iptables/tables.sls

```diff
@@ -4,15 +4,21 @@
 {%- from "iptables/map.jinja" import firewall with context %}
 
 {%- for t in ['filter','nat','mangle'] %}
-  {%- for cn, cv in firewall.get(t)|dictsort %}
+  {%- for cn, cv in firewall.get(t).items() %}
     {%- set pol = cv.policy | default('ACCEPT') %}
     {%- set rules = cv.rules | default({}) %}
 
 chain_present_{{ t }}_{{ cn }}:
   iptables.chain_present:
     - table: {{ t }}
     - name: {{ cn }}
+  {%- endfor %}
+{%- endfor %}
 
+{%- for t in ['filter','nat','mangle'] %}
+  {%- for cn, cv in firewall.get(t).items() %}
+    {%- set pol = cv.policy | default('ACCEPT') %}
+    {%- set rules = cv.rules | default({}) %}
     {%- for rn, rv in rules|dictsort %}
```

Comment:
You should remove `dictsort` here too, right?

Reply:
I actually like it here as it :
1. Does not touch a name of a iptables internal
2. Allows some control over order whilst merging pillar values. ( 02_block_dhcp_requests.sls as default and 01_allow_dhcp_request.sls on server for example